variation/normalize.py

```diff
@@ -57,7 +59,7 @@ def normalize(self, q: str, validations: ValidationSummary,
 
                 variation = valid_result.variation
 
-                variation_id = variation.pop('_id')
+                variation_id = variation['_id']
```

Comment:
double quotes?

edit: looks like this file has only been partially switched to double quotes so ignore if that's a later task

Reply:
Ya i plan on doing that in #185